### DISCUSSION TITLE
Vactubes

### DISCUSSION THREAD

**Discussion by ItzAlmaZ** [BEEmod/BEE2-items#4128] [Tech Support / Q&A] [answered]
What for need this and how i can place tubes normal?
![IMG_2022_04_29_14_10_16_5448401749032213029088](https://user-images.githubusercontent.com/104623098/165916453-9f93b2a9-86f8-4bb4-82ed-95df9478936f.jpg)

**Comment by slambo311** [comment, marked as answer]
1. No offense, but I can tell that English isn't your first language. With proper grammar, it'd be "What do I need this for and how can I place tubes normally?"
2. Place tubes in the path you want them to be in, excluding turns. Then, connect them in order from the start of the path to the end of the path. Make sure to rotate them in the right direction.
3. There are several needs for them, mostly moving players or objects from place to place.

**Reply by ItzAlmaZ** [reply]
Yes, English is not my main language, I need information, why do I need pipes with 9, 4 and 1 square previously called 3x3, 2x2 and 1x1

**Reply by slambo311** [reply]
They control how many squares the turns will take up.
